### ISSUE TITLE
Header text in menu items

### ISSUE THREAD

**Issue by MacNB** [godaddy/wp-escapade-theme#41]
New to WP. Download loaded the this cool Escapade theme but having trouble getting it to work.
Issue is that any Header text I put on any page, ALSO end up twice in the right side bar (one at the top and second under all the menu times).
See picture.

How do I stop that ?

![screen shot 2017-03-07 at 17 48 42](https://cloud.githubusercontent.com/assets/3777537/23670339/47444778-035f-11e7-901f-4fddcbd3f829.png)

**Comment by MacNB** [comment]
So is this an issue or am I doing something wrong ?

**Comment by EvanHerman** [comment]
Hi @MacNB ,

I'm taking a look into the issue now. I'll report back here with some findings shortly.

Thanks!
Evan

**Comment by EvanHerman** [comment]
Hi @MacNB,

I just went through and did some testing on both our GoDaddy Managed WordPress onboarding and by downloading the latest version here on Github and I'm not seeing the error on either instances. Do you have a link to your site for further investigation?

Did you download the theme here on Github and manually upload it to your site?

Thanks,
Evan

**Comment by MacNB** [comment]
Hi EvanHerman,
Thanks for testing this issue. 
No I do not have a link to the site as I am testing it on my iMac using MAMP (i.e. local site).
My godaddy WP managed site is live but uses a custom theme developed by web designers and I do not want to test drive there (I do not have staging).

Yes I downloaded the theme from here and upload it into WP dashboard themes.
What was strange is that when I clicked Add Theme and searching for this theme, WP on my local site does not find it:

![screen shot 2017-03-10 at 23 12 17](https://cloud.githubusercontent.com/assets/3777537/23816983/cf030e76-05e7-11e7-987f-8e6224bba0d2.png)

so I came here to Github and downloaded the zip file and had to manually install the theme.

Before my Godaddy site went live, I did try this theme on my Godaddy account and I don't recall having this problem.

Issue may be my local site, or there's a difference in what's on GitHub and what Godaddy provides.
I have tried deleting this theme and downloading again and made no difference.
I deleted the testbed site and started again by installing WP and down loaded this theme again and still makes makes not difference.

What can I try to test/debug more ?

**Comment by EvanHerman** [comment]
Hi @MacNB,

The reason that the theme doesn't show up when searched for from within the dashboard (or on WordPress.org) is because the theme is not yet hosted on WordPress.org. We are working with the theme review team to get all of our themes accepted and hosted in the theme repository on WordPress.org, but we have to go through each theme one at a time.

Once the themes are reviewed and accepted you should be able to search for them from within the dashboard without any issues. Until then, the themes are hosted here on Github.

When you downloaded the theme here from Github, did you just click the download zip button nested underneath 'Clone or Download'? Do you happen to know what branch it was? Or did you click on 'Releases' and download the latest release from there?


Evan

**Comment by MacNB** [comment]
Hi @EvanHerman 
Thanks for the explanation as to why the this theme does not appear in the search.

Yes I just clicked on the Download zip button. Not sure which was that branch I downloaded. The readme.txt file in the zip says "Stable tag: 1.0.1". This is the screenshot of what I clicked:
![esc-dl](https://cloud.githubusercontent.com/assets/3777537/23862344/a6d27ce2-0804-11e7-9f24-f133fe53b08e.png)

Was that the release version (on 5th march) ?

**Comment by fjarrett** [comment]
@MacNB You should download `escapade.zip` from the release area, rather than the entire repo which contains lots and lots of development files.

https://github.com/godaddy/wp-escapade-theme/releases

Also be sure to have the latest version of the Primer parent theme, currently 1.6.0.

**Comment by EvanHerman** [comment]
Hi @MacNB,

Thank you for clarifying that. I went ahead and just did the same - headed over to https://github.com/godaddy/wp-escapade-theme/ and clicked 'Clone or download' and clicked 'Download ZIP'.

This downloaded a .zip file from the 'develop' branch named `wp-escapade-theme-develop.zip`.

I then went ahead and uploaded that to my test instance, activated it and viewed a test page - and actually got the same results as you did.

![Example](https://cloudup.com/cSDQTtioel6)

Following that, I went and tested the latest release from the releases tab (v1.0.1) https://github.com/godaddy/wp-escapade-theme/releases -  you'll want to click the **escpade.zip** link just under the v1.0.1 latest release.

From there you can then upload those files to your site, and remove the existing develop branch files.

I noticed that using the v1.0.1 release works just fine:

![example](https://cloudup.com/c0o-4ml4Z4x)

Finally, as a last test I went back and just changed the directory name on the original theme files from `wp-escpade-theme-develop` to `escapade` and re-activated the theme. Viola, no issues any longer.

In short: Try updating the theme directory name from `wp-escpade-theme-develop` to `escapade` to see if that helps alleviate the issue. If not, go ahead and download the latest release (v1.0.1) from the releases tab, and re-upload it to your site which should ultimately resolve the issue.

Let us know if that helps resolve the issue on your end as well.


Evan

**Comment by MacNB** [comment]
@EvanHerman and @fjarrett thanks for your suggestions.
Will try it when I get home and update the issue.

**Comment by MacNB** [comment]
Ok. Issue fixed by using the Released escapade.zip as opposed to using the cloned download.
So implies the current git download is "work-in-progress" and not stable ?

PS. apologies for late reply but work got in the way :)

**Comment by EvanHerman** [comment]
Hi @MacNB ,

No worries - we're glad you were able to get things up and running.

Correct, the best place to get the most stable release is via the 'Releases' tab at the top of the repository. Alternatively, you could download it the way you did but you would have to ensure it was off of the `master` branch, and that when you unzipped it you renamed the directory appropriately.

It's just easier to download it from the releases tab, and that way you know you are getting the most up to date, stable, release of the theme.

Thanks again!
Evan

**Comment by MacNB** [comment]
@EvanHerman ,
Many thanks for your tips and support.
